force-app/main/default/lwc/graphqlContacts/graphqlContacts.js

```diff
@@ -7,7 +7,11 @@ export default class GraphqlContacts extends LightningElement {
             query getContacts {
                 uiapi {
                     query {
-                        Contact(first: 5, orderBy: { Name: { order: ASC } }) {
```

Comment:
We can't add these changes to `graphqlContacts` yet because of a Platform deployment bug. See the output of CI for PR #816:
https://github.com/trailheadapps/lwc-recipes/actions/runs/5276425559/jobs/9760703283?pr=816